tests/foreman/ui/test_discoveryrule.py

```diff
@@ -228,9 +245,8 @@ def test_positive_list_host_based_on_rule_search_query(
         host_name = values['table'][0]['Name']
         assert values['searchbox'] == f'discovery_rule = "{discovery_rule.name}"'
         assert len(values['table']) == 1
-        assert provisioned_host_name in host_name
```

Comment:
Any reason to remove this assertion?

Reply:
this assertation was useless since provisioned_host_name is a domain name which we check assert is present in host_name